### ISSUE TITLE
Salva múltiplos usuários e endereços (repetidos)

### ISSUE THREAD

**Issue by mulleranjos** [deivisonarthur/OSC-Magento-Brasil#71]
Não sei se alguns de vocês já tiveram este problema:

Você faz o cadastro como um novo cliente. Cadastra um endereço A. Faz o finaliza o checkout e inicia uma nova compra, ainda logado. Porém, dessa vez, durante o checkout, você escolhe para adicionar um novo endereço B. Finaliza o checkout com o endereço B. A partir desse momento, sempre que o usuário começa fazer o checkout, com o endereço A ou B (independente), ele começa a regravar múltiplas vezes o endereço A ou B, a cada checkout, como se fosse um endereço novo.

Fiz a instalação do IWD Checkout e percebi que esse problema não ocorre.

A sensação que tenho é que o problema está na recepção dos dados e gravação no BD, e não na página de billing.phtml. 

Alguém tem alguma idéia de como resolver isso? 

Att.

Bruno

**Comment by zepafonso** [comment]
Estou tendo exatamente o mesmo problema!

**Comment by vinidade** [comment]
Eu também estou com esse problema.

**Comment by mulleranjos** [comment]
Alguém poderia dar uma luz?

**Comment by JoseEduardo** [comment]
Alguém conseguiu resolver esse problema? já faz bastante tempo e ninguém arrumou esse Bug...

**Comment by flavioea** [comment]
Sim, ele ainda existe. Mas apenas nestas condições que o zepafonso falou...
E se vc apagar os endereços duplicados e fazer uma compra usando um local para faturamento e outro para entrega, ele não retorna o erro...

Só há uma nova duplicação se vc usar entrega alternativa... enfim.. tá bagunçado mesmo!

Um detalhe q eu notei: no admin nao aparece os endereços.. apenas o principal meeesmo!
